.editorconfig

```diff
@@ -5,8 +5,16 @@ root = true
 
 # Unix-style newlines with a newline ending every file
 [*]
+charset = utf-8
 end_of_line = lf
+indent_style = space
+indent_size = 4
 insert_final_newline = true
+trim_trailing_whitespace = true
+
+# GO files
+[*.go]
+indent_style = tab
```

Comment:
Nice that you can do this specifically by language. 

Of course... php is yet another world, and we have a few PHP files; not sure we have to deal with it though?

Reply:
That's now handled properly with the default config above. By default spaces are used and exceptions are provided later for `*.go` and `Makefile`. Maybe we have to define other exception I don't see at the moment.